README.md

```diff
@@ -212,6 +211,51 @@ _Notes:_
 
 Compilation to iOS is supported on macOS targeting the iOS simulator (`--target x86_64-apple-ios`) and 64 bit ARM devices (`--target aarch64-apple-ios`). The ARM64**e** architecture is [not supported yet](https://github.com/rust-lang/rust/issues/73628).
 
+### For WebAssembly
+
+Install `emscripten` version 3.1.1 or superior.
+
+Build with the `wasm32-unknown-emscripten` target (`wasm32-unknown-unknown` is
+unsupported because it is [fundamentally incompatible with linking C code](https://github.com/rustwasm/team/issues/291#issuecomment-645482430):
+
+```bash
+export EMSDK=~/.asdf/installs/emsdk/3.1.1
```

Comment:
In the wasm-example's README.md it's mentioned that the environment paths assume that emscripten got installed through the asdf plugin. I think it would be helpful to mention this here, too.

Reply:
Done in fb596026